packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -77,6 +79,7 @@ interface ImagePixelModule {
   pixelRepresentation: string;
   windowWidth: number;
   windowCenter: number;
+  voiLUTFunction: string;
```

Comment:
Can we use the Enum you created instead of `string` here as the type as well? and you handle the metadata reading and assignging the correct Enum?